noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -273,7 +273,7 @@ impl BaseRollupInputs {
                 );
 
                 let new_value = compute_public_data_tree_value(existing_update.new_value - tx_fee);
-                let protocol_update_request = PublicDataUpdateRequest { leaf_slot, new_value };
+                let protocol_update_request = PublicDataUpdateRequest { leaf_slot, new_value, counter: 0 };
```

Comment:
It's probably worth a comment in the code, but by the time we're in the base rollup, we don't care about the counters, which is why we arbitrarily set it to 0.